futurex_openedx_extensions/dashboard/statistics/learners.py

```diff
@@ -195,7 +196,7 @@ def get_learners_count(fx_permission_info: dict) -> Dict[int, Dict[str, int]]:
     }
     for tenant_id in fx_permission_info['permitted_tenant_ids']:
         result[tenant_id]['learners_count_per_org'] = {
-            item['org']: item['learners_count']
+            item['org_upper_case'].upper(): item['learners_count']
```

Comment:
```suggestion
            item['org_lower_case'].lower(): item['learners_count']
```

Reply:
done, everywhere. Even in tests, to keep consistency and clear readability. But I left some test-data in upper-case which helps to detect failing code. Nothing is failing 🥳